src/core/packages/apps/browser-internal/src/status/components/status_table.test.tsx

```diff
@@ -8,7 +8,9 @@
  */
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mountWithIntl } from '@kbn/test-jest-helpers';
```

Comment:
This library uses `enzyme` under the hood. @Dosant, are we planning to migrate `@kbn/test-jest-helpers` to RTL?

Reply:
I am not sure how this will look like, but we likely won't be able to internally migrate this helper without touching the actual test case code.
We should probably create a separate package for most common react testing library cases (if doesn't exist yet) and deprecate these functions. 

For now, I'd either use react testing library directly or this wrapper 

https://github.com/elastic/kibana/blob/6a7c904f921434fe21dfa00eceabfb5e64e915dc/src/platform/packages/shared/kbn-test-jest-helpers/src/testing_library_react_helpers.tsx#L21-L25

Reply:
@Bamieh, @TinaHeiligers, 

> LGTM we should be using @kbn/test-jest-helpers anyways since it has the custom centralized code written by the core team for testing i18n

I don't think we should. 
I don't think we will be able to migrate mountWithIntl internally to use testing library without rewriting the test cases. 

imo, mountWithIntl should be marked as deprecated just us any other enzyme api and another common util needs to be used